extensions/binding/org.eclipse.smarthome.binding.wemo/ESH-INF/thing/thing-types.xml

```diff
@@ -144,6 +145,13 @@
         <state readOnly="true"></state>
     </channel-type>
     
+     <channel-type id="lastMotionDetected" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Last Activity (Date/Time)</label>
+        <description>Date/time when last motion was detected</description>
+        <category>Date</category>
```

Comment:
Well, Date isn't even listed [here](https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channel-categories), therefore it is clearly wrong to put it here.